doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -156,12 +151,14 @@ Considering that different vendors platform may have different LED color capabil
 
 ## 2. System health monitor service business logic
 
-System health monitor daemon will running on the host, periodically(every 60s) check the "monit summary" command output and PSU, fan, thermal status which stored in the state DB, if anything wrong with the services monitored by monit or peripheral devices, system status LED will be set to fault status. When fault condition relieved, system status will be set to normal status. 
+System health monitor daemon will running on the host, periodically(every 60s) check critical services/processes status, the "monit summary" command output and PSU, fan, thermal status which stored in the state DB, if anything wrong with them, system status LED will be set to fault status. When fault condition relieved, system status will be set to normal status.
 
-Before the switch boot up finish, the system health monitoring service shall be able to know the switch is in boot up status(see open question 1).
+Before the switch boot up finish, the system health monitoring service shall get the monit service startup delay and make sure monit service run first.
 
-If monit service is not avalaible, will consider system in fault condition.
-FAN/PSU/ASIC data not available will also considered as fault conditon.
+Empty FEATURE table will be considered as fault condition.
```

Comment:
How do we define the term `fault condition`? This term appears frequently in this document.

Reply:
`fault condition` means that the system status is bad, and system health service shall put an error log to syslog and mark the overall system status as "Not OK".